composer.json

```
@@ -19,7 +19,7 @@
     ],
     "require": {
         "neos/neos": "~4.2 || ~5.0 || ~7.0 || dev-master",
-        "sebastian/diff": "3.*"
+        "sebastian/diff": "4.*"
```

Why not allow both to prevent other conflicts?

```suggestion
        "sebastian/diff": "3.* || 4.*"
```